cmd/keycloakb/keycloak_bridge.go

```diff
@@ -532,6 +532,8 @@ func main() {
 	}
 	var glnVerifier = business.NewGlnVerifier(glnLookupProviders...)
 
+	profile.GlnVerifier = glnVerifier
```

Comment:
I'm not fan of static assigning variables from other packages.
You should pass glnVerifier to each "object" you are creating and which will need it

Reply:
After talking to Francis, we concluded that the current solution was fine.